packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -28,6 +28,13 @@ export class Popup extends BaseComponent<IPopupProps, IPopupState> {
     this._originalFocusedElement = getDocument()!.activeElement as HTMLElement;
   }
 
+  public shouldComponentUpdate(nextProps: IPopupProps): boolean {
```

Comment:
Overriding shouldComponentUpdate is generally a bad idea since it can lead to bugs. In fact, the react docs discourage it: https://reactjs.org/docs/react-component.html#shouldcomponentupdate

In most React apps, folks use "PureComponent" to implement shouldComponentUpdate. I think that might be hard in fabric since everything implements base component, but I think moving towards leaning on PureComponent here would be better than manually passing through a skipUpdate prop.